recipes/parg/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

Comment:
```suggestion
conan_basic_setup(TARGETS)
```

Reply:
not sure if it's important as well, as we use neither `CONAN_LIBS` nor `CONAN_PKG::` here.
`conan_basic_setup` is only needed to set correct flags.

Reply:
it's important to really test your CMake target.